Traducir.Core/Services/SOStringService.cs

```diff
@@ -596,5 +597,30 @@ Update Strings
             }
             return _strings;
         }
+
+        public async Task<bool> DeleteSuggestionAsync(SOStringSuggestion suggestion)
+        {
+            using (var db = _dbService.GetConnection())
+            {
+                await db.ExecuteAsync(@"
+Update StringSuggestions
+Set StateId = {=DeletedByOwner}
+Where Id = @suggestionId;
+
+Insert Into StringSuggestionHistory
+            (StringSuggestionId, HistoryTypeId, UserId, Comment, CreationDate)
+Values      (@suggestionID, {=DeletedByOwner}, @userId, @comment, @now);",
+                new
+                {
+                    suggestionId = suggestion.Id,
+                    StringSuggestionState.DeletedByOwner,
+                    userId = suggestion.CreatedById,
+                    comment = "Deleted by user",
```

Comment:
leave the comment on null if it doesn't add anything else. The `HistoryType` already tells us that it was deleted by the owner.

I'm using the comment to provide context for an action (for instance, if a suggestion was discarded because another was approved, I'm adding the id of the approved one on the comment)